<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Regist</class>
 <widget class="QWidget" name="Regist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background:rgb(255, 255, 255);
font-size:15px;
font-style:MingLiU-ExtB;
}
QFrame{
border:sold 10px rgba(0,0,0);
background-image: url(:/assets/loginbackground.png);
}
QLineEdit{
color:rgb(255, 255, 255);
background-color:rgba(64, 83, 97, 230);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QLineEdit:hover{
color:rgb(255, 255, 255);
background-color:rgba(64, 83, 97, 80);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QLineEdit:focus{
color:rgb(255, 255, 255);
background-color:rgba(64, 83, 97, 180);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}

QComboBox{
color:rgb(255, 255, 255);
background-color:rgba(64, 83, 97, 230);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QComboBox:hover{
color:#8d98a1;
background-color:rgba(64, 83, 97, 80);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QComboBox:focus{
color:#8d98a1;
background-color:rgba(64, 83, 97, 180);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}

QPushButton{
background:rgb(65, 175, 235);
color:rgb(220, 220, 220);
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton:hover{
background-color:rgba(80, 190, 250,130);
border-style:inset;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
background-color:rgba(80, 190, 250,255);
border-style:inset;
font-style:MingLiU-ExtB;
}



QPushButton#hideButton{
background: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QPushButton#hideButton:hover{
background: rgba(212, 212, 212, 100);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QPushButton#hideButton:pressed{
background: rgba(212, 212, 212, 255);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QPushButton#closeButton{
background: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QPushButton#closeButton:hover{
background: rgba(255, 0, 0, 100);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QPushButton#closeButton:pressed{
background: rgba(255, 0, 0, 255);
color: rgb(255, 255, 255);
font-style:MingLiU-ExtB;
}
QCheckBox{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>731</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string extracomment="无功能label，可以换成其他字或者图片">WeNoChat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>630</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hideButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>—</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>注册新用户</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>重复密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneNumberEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>手机号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>密保问题回答</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>passwordEdit_2</tabstop>
  <tabstop>phoneNumberEdit</tabstop>
  <tabstop>answerEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>hideButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
